news/2025/2025-03-25-the-followpoint-special-behind-the-banhammer.md

```diff
@@ -0,0 +1,292 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Speaking of which, I've heard of GMT applications opening for people who speak a specific language in the past, to get more people for channels that lack moderation, but how and who usually decides that a specific language needs more moderators?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/5999631" style="background-image: url('/wiki/shared/avatars/Venix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/5999631">Venix</a></p>
+
+Usually, it's just somebody from the team pointing out the critical lack of moderators in a certain area; you know — it's not that hard to notice when things are getting out of hand somewhere.
+
+For regular day-to-day additions it usually works the opposite way: if we have a person be active in a channel or a certain area, their candidature will be likely discussed when they are here rather than specifying a need for moderation of a certain field and then considering the addition.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Do you have any plans to have a permanent application system in the future, like what BNs have on their website?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3227533" style="background-image: url('/wiki/shared/avatars/Kobold84.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/3227533">Kobold84</a></p>
+
+The short answer is we don't have "plans" to have them permanent.
+
+The longer answer is that we've had this suggestion from time to time, and it seems enticing, since the workload is huge if we go through all applicants at once. However, this suggestion hasn't had a lot of support, and one of the main reasons I can think of is that it's important for our team to have a huge influx of "fresh blood" to lessen the disconnect between the community and the GMT, this way there's a natural rotation and no stagnant air. From my experience with all of the apps it does seem true, with each new round iteration you could see positive changes within the team after bringing in dozens of new members all at once. Hopefully, the upcoming apps will work out the same way.
```

Comment:
```suggestion
The longer answer is that we've had this suggestion from time to time, and it seems enticing, since the workload is huge if we go through all applicants at once. However, this suggestion hasn't had a lot of support, and one of the main reasons I can think of is that it's important for our team to have a huge influx of "fresh blood" to lessen the disconnect between the community and the GMT. This way there's a natural rotation and no stagnant air.

From my experience with all of the apps it does seem true that with each new round, you could see positive changes within the team after bringing in dozens of new members all at once. Hopefully, the [current apps](https://osu.ppy.sh/home/news/2025-03-09-gmt-apps-now-open) will work out the same way.
```